Makefile

```diff
@@ -97,6 +100,10 @@ modules-docs: chpldoc
 # dependency so parallel make executions correctly build chpldoc first.
 	$(MAKE) modules-docs-only
 
+module-docs: modules-docs
```

Comment:
I'd like to replace it too.  I kept typing the wrong thing every time I tried this.  Ditto modules-docs-only => module-docs-only.